rosbag2_storage_mcap/CMakeLists.txt

```diff
@@ -43,6 +44,7 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE "ROSBAG2_STORAGE_MCAP_BUILDIN
 target_link_libraries(${PROJECT_NAME}
   mcap_vendor::mcap
   pluginlib::pluginlib
+  rclcpp::rclcpp
```

Comment:
@pfavr2 Have I missed something?  
Why do we need `rclcpp` package dependencies on the storage plugins layer?